metricbeat/module/kubernetes/_meta/test/docs/without-ksm/metricbeat-kubernetes.yaml

```diff
@@ -187,6 +177,9 @@ spec:
               subPath: metricbeat.yml
             - name: data
               mountPath: /usr/share/metricbeat/data
+            - name: kibana
```

Comment:
Out of curiosity, why is this needed?

Reply:
I must confess I did not check the files for testing. It was not the purpose of this PR, but I can check them later